usegalaxy.org/beacon.yml

```diff
@@ -0,0 +1,26 @@
+tool_panel_section_label: Beacon
```

Comment:
There doesn't seem to be a section like this on usegalaxy.eu ? Can you add this to the `Variant Calling` section please.